README.md

```diff
@@ -269,15 +269,56 @@ See [dht-rpc](https://github.com/mafintosh/dht-rpc) for the additional APIs the
 
 ## CLI
 
-You can start a DHT node in the command line, with the bundled cli tool:
+You can start a DHT node in the command line:
 
 ```sh
 npm install -g @hyperswarm/dht
-hyperswarm-dht # runs a DHT node
-hyperswarm-dht --bootstrap # runs a DHT node with bootstrap settings
-hyperswarm-dht --nodes 5 # runs 5 nodes
 ```
 
+Run a DHT node:
+```sh
+hyperswarm-dht # [--port 0] [--host 0.0.0.0] [--bootstrap <comma separated list of ip:port>]
+```
+
+Or run multiple nodes:
+```sh
+hyperswarm-dht --nodes 5 # [--host 0.0.0.0] [--bootstrap <list>]
+```
+
+Note: by default it uses the [mainnet bootstrap nodes](lib/constants.js).
+
+#### Isolated DHT network
+
+To create your own DHT network is as follows:
+
+1) Run your first bootstrap node:
+```sh
+hyperswarm-dht --bootstrap --host (server-ip) # [--port 49737]
+```
+
+Important: it requires the port to be open.
+
+Now your bootstrap node is ready to use at `(server-ip):49737`, for example:
+```js
+const node = new DHT({ bootstrap: ['(server-ip):49737'] })
+```
+
+Note: You could configure some DNS for the bootstrap IP addresses.
+
+For the network to be fully operational it needs at least one persistent node.
+
+2) Provide the first persistent node using your own bootstrap values:
+
+```sh
+hyperswarm-dht --port 49738 --bootstrap (server-ip):49737 --persistent
+```
+
+Important: it requires the port to be open like the bootstrap node.
+
+Don't use the `--persistent` flag again, or you could damage the network health.
```

Comment:
lets not have the flag, footgun

Reply:
100%, just depends on https://github.com/mafintosh/dht-rpc/pull/58